src/webui/webapplication.cpp

```diff
@@ -262,12 +262,12 @@ void WebApplication::action_public_images()
 void WebApplication::action_query_torrents()
 {
     CHECK_URI(0);
-    const QStringMap& gets = request().gets;
 
+    const auto gets = [&](const char *field) -> QString { return request().gets[field]; };
```

Comment:
Why these changes? Why do you need this extra call now?

Reply:
reverted.